examples/cookbook/design/drawer/lib/drawer.dart

```diff
@@ -3,6 +3,7 @@ import 'package:flutter/material.dart';
 void drawerStart() {
   // #docregion DrawerStart
   Scaffold(
+    appBar: AppBar(), // Add an AppBar with hamburger button.
```

Comment:
I'd add a `title` to the `AppBar` to make it obvious where it is. Something like 'AppBar without hambuger button'.

The comment is misleading, at this point it will **not** have a hambuger button.

Reply:
Well, only `AppBar()` is enough to show the hamburger button. I added a title as you suggested but I used the text 'AppBar **with** hamburger button' .

Reply:
On android:

<img src="https://github.com/flutter/website/assets/31314944/1fbaac3e-feff-4f47-873a-2366812c6890" width="300px" alt="screenshot" />